src/plugin/main.c

```diff
@@ -342,6 +342,9 @@ static obs_properties_t* browser_get_properties(void* vptr)
 	                        "*.so", NULL);
 	obs_properties_add_text(props, "flash_version", obs_module_text("FlashVersion"),
 	                        OBS_TEXT_DEFAULT);
+	obs_properties_add_editable_list(props, "cef_environment",
```

Comment:
You might want to consider looking for a way to store a set of key-value-pairs in obs properties, but my guess is that these don't exist.

Reply:
Unfortunately, I don't think there is such a way. At least I don't know about any ways to open alert windows or such to inform a user about illegal input in OBS.